app/livechat/server/api/v1/message.js

```diff
@@ -95,9 +96,9 @@ API.v1.addRoute('livechat/message/:_id', {
 				throw new Meteor.Error('invalid-message');
 			}
 
-			return API.v1.success({ message });
+			return API.v1.success({ message: normalizeMessageAttachments(message) });
```

Comment:
Do you mind of checking the `message.file` before returning? I think we should always return the `message` instead of the `normalizeMessageAttachments`.